api/controllers/game/rematch.js

```diff
@@ -0,0 +1,64 @@
+const gameService = require('../../services/gameService');
+
+/**
+ * Indicate that the player wants to play again
+ */
+const gameAPI = sails.hooks['customgamehook'];
+module.exports = async function (req, res) {
+  try {
+    const { game: gameId, pNum } = req.session;
+    const game = await Game.findOne({ id: gameId }).populate('players');
+    const gameUpdates = {};
+    if (pNum === 0 || pNum === '0') {
+      gameUpdates.p0Rematch = true;
+    } else {
+      gameUpdates.p1Rematch = true;
+    }
+
+    await Game.updateOne({ id: game.id }).set(gameUpdates);
```

Comment:
```suggestion

    await Game.updateOne({ id: game.id }).set({`[p${pNum}Rematch`]:true});
```

Reply:
Upon further thought I think we could break this down even further to a one-liner, unless there is a reason to explicitly show what's happening.

Reply:
`gameUpdates` is referenced later in this block so I'll declare it explicitly